### PR TITLE
Update translations for 15.2 release

### PR DESCRIPTION
**Pull request by def-**
## Checklist

- [ ] Tested the change ingame
- [ ] Provided screenshots if it is a visual change
- [ ] Tested in combination with possibly related configuration options
- [ ] Written a unit test if it works standalone, system.c especially
- [ ] Considered possible null pointers and out of bounds array indexing
- [ ] Changed no physics that affect existing maps
- [ ] Tested the change with [ASan+UBSan or valgrind's memcheck](https://github.com/ddnet/ddnet/#using-addresssanitizer--undefinedbehavioursanitizer-or-valgrinds-memcheck) (optional)

### CODE REVIEW

File: data/languages/brazilian_portuguese.txt [hunk 1]
```diff
@@ -1284,3 +1266,42 @@ Particles
...
 Assets directory
 == Diretório de ativos
+
+Debug mode enabled. Press Ctrl+Shift+D to disable debug mode.
+== 
+
+Existing Player
+== 
+
+Your nick name '%s' is already used (%d points). Do you still want to use it?
```
Comment (b3z) [inline data/languages/brazilian_portuguese.txt:1276]: nickname is one word.

File: data/languages/bulgarian.txt [hunk 2]
```diff
@@ -703,6 +703,15 @@ It's recommended that you check the settings to adjust them to your liking befor
 Please enter your nick name below.
 == 
 
+Existing Player
+== 
+
+Your nick name '%s' is already used (%d points). Do you still want to use it?
```
Comment (b3z) [inline data/languages/bulgarian.txt:709]: again, nickname is one word. Same like above.

File: data/languages/catalan.txt [hunk 3]
```diff
@@ -1170,9 +1155,21 @@ The width or height of texture %s is not divisible by 16, which might cause visu
...
+Debug mode enabled. Press Ctrl+Shift+D to disable debug mode.
+== 
+
 Use k key to kill (restart), q to pause and watch other players. See settings for other key binds.
 == 
 
+Existing Player
+== 
+
+Your nick name '%s' is already used (%d points). Do you still want to use it?
```
Comment (b3z) [inline data/languages/catalan.txt:1167]: nickname

File: data/languages/belarusian.txt [hunk 4]
```diff
@@ -706,6 +706,15 @@ It's recommended that you check the settings to adjust them to your liking befor
 Please enter your nick name below.
```
Comment (b3z) [inline data/languages/belarusian.txt:706]: nickname, okay you get it. its 49 results. won't comment all.

**Review by b3z — CHANGES_REQUESTED** [review]
nickname is one word.

File: data/languages/brazilian_portuguese.txt (hunk 1 above)
Comment (b3z) [reply data/languages/brazilian_portuguese.txt:1276]: 48 more cases

Comment (def-) [reply data/languages/brazilian_portuguese.txt]: Done.

**Review by b3z — APPROVED** [review]
👍

**Pull request merged**